pkg/search/result.go

```diff
@@ -68,6 +95,7 @@ type Repository struct {
 	PushedAt        time.Time `json:"pushed_at"`
 	Size            int       `json:"size"`
 	StargazersCount int       `json:"stargazers_count"`
+	URL             string    `json:"url"`
```

Comment:
I think we'd want to use the `html_url` JSON property here. The `url` property is the REST API url, which I don't think is useful to display in this context.